pkg/nsx/endpoint.go

```diff
@@ -174,19 +174,19 @@ func (ep *Endpoint) KeepAlive() {
 }
 
 func (ep *Endpoint) setup() {
-	log.V(1).Info("begin to setup endpoint")
+	log.V(2).Info("begin to setup endpoint")
 	err := ep.keepAlive()
 	if err != nil {
 		log.Error(err, "setup endpoint failed")
 	} else {
-		log.V(1).Info("setup endpoint successfully")
+		log.V(2).Info("setup endpoint successfully")
```

Comment:
line 177,182 setup normally run once, it's better to set level to 1

Reply:
ok